lib/exports.js

```diff
@@ -98,15 +98,15 @@ function Exports(api, logger) {
 
       var attempt = function attempt() {
         return _this.findById({ auditId: auditId, id: id }).then(function (response) {
-          if (response.status === 'SUCCESS') {
-            return response;
-          } else if (response.status === 'IN PROGRESS' && attempts < tries) {
+          if (response.status === 'FAILED') return _bluebird2.default.reject(response.error);
+          if (response.status === 'SUCCESS') return response;
+          if (response.status === 'IN PROGRESS' && attempts < tries) {
             return _bluebird2.default.delay(poll).then(function () {
               attempts += 1;
               return attempt();
             });
           }
-          return _bluebird2.default.reject(new Error('Request to get export timed out. Increasing \'tries\' may help. Failed after ' + attempts + ' tries.'));
+          return _bluebird2.default.reject(new Error('Request to get export timed out after ' + attempts + ' attempts.'));
```

Comment:
Should use backticks here, same as you did in the changes below

Eg: `Request to get export timed out after ${attempts} attempts`